widgets/ScalaSparkCompute-sparkcompute.json

```diff
@@ -1,6 +1,6 @@
 {
   "metadata": {
-    "spec-version": "1.5"
+    "spec-version": "1.4"
```

Comment:
this should be 1.5 I think. ``display-name`` and ``icon`` are supported in 1.5.

Reply:
ajai mentioned it is 1.4. But it seems like these versions really haven't needed to be incremented at all. There are no backwards incompatibilities and new features not present in older specs work anyway.